src/Compiler/CompilerUtils.cpp

```diff
@@ -251,6 +237,27 @@ struct Command {
 // =============================================================================
 // Methods for compiling and file processing.
 
+std::string getOnnxMlirFullVersion() {
+  const std::string OnnxMlirVersion = "onnx-mlir version 1.0.0";
+  return
+#ifdef ONNX_MLIR_VENDOR
+      ONNX_MLIR_VENDOR ", " + OnnxMlirVersion;
+#elif defined(ONNX_MLIR_REPOSITORY) && defined(ONNX_MLIR_REVISION) &&          \
+    defined(LLVM_REPOSITORY) && defined(LLVM_REVISION)
+  // Intended for code review, and will remove one option before merging.
```

Comment:
Let's decide if we want to simplify the version output to 
1. no repository and short revision
2. or, repository and full revision

Reply:
@cjvolzka Do you agree to use `-DPACKAGE_VENDOR` for no repository and short version?

Reply:
+1 for `onnx-mlir version 1.0.0 (onnx-mlir fd6a145f7702, llvm-project 9778ec057cf4)` which includes LLVM and is clean and neat.

Reply:
+2 for option (1)

Reply:
I agree with @sstamenova and I am also in favor with being consistent with what LLVM does (given also we print out the LLVM commit already).

Reply:
During offline discussion with @gongsu832, we noticed that clang sometimes doesn't output both the repo url and hash, e.g., 
```
clang version 15.0.0
Target: powerpc64le-unknown-linux-gnu
Thread model: posix
InstalledDir: <path to installed dir>
```
It appears to be controlled by `LLVM_APPEND_VC_REV`, I have created PR https://github.com/onnx/onnx-mlir/pull/1446 to add a similar logic. 
@gongsu832 was amenable of using option (2).